src/main/java/org/jpeek/App.java

```diff
@@ -171,123 +170,122 @@ public void analyze() throws IOException {
         final XSL chain = new XSLChain(layers);
         this.save(skeleton.toString(), "skeleton.xml");
         final Collection<Report> reports = new LinkedList<>();
-        final Calculus xsl = new XslCalculus();
         if (this.params.containsKey("LCOM")) {
```

Comment:
@vzurauskas Why not use an iteration on snippet :
```java
if (this.params.containsKey(code)) {
    reports.add(
        new XslReport(
            chain.transform(skeleton), new XslCalculus(code),
             new ReportData(code, this.params, mean, sigma)
        )
    );
}
```
with `code`, `mean` and `sigma` some parameters. You could use `mean` and `sigma` values where it appears only one parameter in `ReportData` constructor. You can also create a `todo` for that if you want.

Reply:
@baudoliver7 That is a good suggestion, however this PR has been blocked for a very long time and I no longer have the time to work on it (at least in the near future). You can take it over if you want.

Reply:
@vzurauskas Ok. But how can I do that ?

Reply:
@baudoliver7 You can ask @paulodamaso to assign this task to you and then copy this branch.